seed/urls.py

```diff
@@ -4,3 +4,13 @@
 :copyright (c) 2014 - 2017, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
 :author
 """
+
```

Comment:
This file was moved from `seed/urls/__init__.py`.  Does that mean there is no longer an __init__.py file there?  Maybe I'll find out as I read along.